src/components/Hero.js

```diff
@@ -2,24 +2,25 @@ import React, { useState } from "react";
 import styled from "styled-components";
 import { IoMdCloseCircle } from "react-icons/io";
 
-const close = (isVisible, setIsVisible) => {
-  setIsVisible(!isVisible);
+const close = (isClosed, setIsClosed) => {
+  localStorage.setItem('verifactHeroClosed', true);
```

Comment:
After considering this, I think a better approach is to serialize and deserialize these values to JSON, allowing you to use real JavaScript boolean values reliably. e.g. `JSON.parse("true") === true`. You could abstract localStorage to a `storage` util and create 2 methods `set` and `get` which serialize and deserialize respectively.

Reply:
Changed in `e8488b6`